session.go

```diff
@@ -283,6 +283,18 @@ func (s *Session) init() error {
 	return nil
 }
 
+// AwaitSchemaAgreement will query the state of the schema versions across
+// all nodes of the cluster. The maximum amount of time this takes is governed
+// by the MaxWaitSchemaAgreement setting in the configuration (default: 60s).
```

Comment:
The documentation comment could be clarified  a little bit. `will query the state of the schema versions` seems to me like it could query just once, but the function retries in the case schema versions do not match.

```
// AwaitSchemaAgreement will wait until schema versions across all nodes in the
// cluster are the same (as seen from the point of view of the control connection).
// The maximum amount of time this takes is governed
// by the MaxWaitSchemaAgreement setting in the configuration (default: 60s).
// AwaitSchemaAgreement returns an error in case schema versions are not the same
// after the timeout specified in MaxWaitSchemaAgreement elapses.
```

Reply:
That's a very good point! I used your suggestion. 💃